# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: build/win32/orderfile.txt
# Commit: 2adbab502a1e
# Full Hash: 2adbab502a1e690fccd61fb604be820e60d61723
# Author: Kristen Wright <kwright@mozilla.com>
# Date: 2020-09-09 09:39:57
# Regressor Bug: 1642727
# File Overlap Count: 1
# Description:
#   Bug 1642727 - Remove the VarCache machinery r=njn
#   
#   This commit removes Varcache entirely, putting an end to our quest to get rid of VarCache.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D79538
# ==============================================================================

diff -r 1aeb7707569f -r 2adbab502a1e build/win32/orderfile.txt
--- a/build/win32/orderfile.txt	Tue Sep 08 15:05:06 2020 +0000
+++ b/build/win32/orderfile.txt	Tue Sep 08 17:12:01 2020 +0000
@@ -465,11 +465,9 @@
 ?AddRef@AccessibleCaretEventHub@mozilla@@UAGKXZ
 ??$AppendElement@AAV?$nsTAutoStringN@D$0EA@@@UnsTArrayInfallibleAllocator@@@?$nsTArray_Impl@V?$nsTString@D@@UnsTArrayInfallibleAllocator@@@@IAEPAV?$nsTString@D@@AAV?$nsTAutoStringN@D$0EA@@@@Z
 ?GetPreference@Preferences@mozilla@@SAXPAVPref@dom@2@@Z
-??$AddAtomicBoolVarCache@$0A@@Preferences@mozilla@@SAXPAV?$Atomic@_N$0A@X@1@ABV?$nsTSubstring@D@@_N@Z
 ??$RegisterCallbackImpl@$$CBV?$nsTSubstring@D@@@Preferences@mozilla@@CA?AW4nsresult@@P6AXPBDPAX@ZABV?$nsTSubstring@D@@1W4MatchKind@01@_N@Z
 ?GetComplex@Preferences@mozilla@@SA?AW4nsresult@@PBDABUnsID@@PAPAXW4PrefValueKind@2@@Z
 ?GetBool@Preferences@mozilla@@SA?AW4nsresult@@PBDPA_NW4PrefValueKind@2@@Z
-??$AddAtomicBoolVarCache@$01@Preferences@mozilla@@SAXPAV?$Atomic@_N$01X@1@ABV?$nsTSubstring@D@@_N@Z
 ?SetPreference@Preferences@mozilla@@SAXABVPref@dom@2@@Z
 ?Allocate@ArenaChunk@?$ArenaAllocator@$0BAAA@$00@mozilla@@QAEPAXI@Z
 ?SavePrefFileBlocking@Preferences@mozilla@@QAE?AW4nsresult@@XZ
@@ -6465,7 +6463,6 @@
 ?ComputeCrossOriginOpenerPolicy@HttpBaseChannel@net@mozilla@@UAG?AW4nsresult@@W4CrossOriginOpenerPolicy@nsILoadInfo@@PAW456@@Z
 ?OnStartRequest@nsDocumentOpenInfo@@UAG?AW4nsresult@@PAVnsIRequest@@@Z
 ?GetResponseStatus@HttpBaseChannel@net@mozilla@@UAG?AW4nsresult@@PAI@Z
-?AddBoolVarCache@Preferences@mozilla@@SAXPA_NABV?$nsTSubstring@D@@_N@Z
 ?GetResponseHeader@HttpBaseChannel@net@mozilla@@UAG?AW4nsresult@@ABV?$nsTSubstring@D@@AAV5@@Z
 ?GetHeader@nsHttpRequestHead@net@mozilla@@QAE?AW4nsresult@@UnsHttpAtom@23@AAV?$nsTSubstring@D@@@Z
 ?GetStatus@HttpBaseChannel@net@mozilla@@UAG?AW4nsresult@@PAW44@@Z